test/recipe/admin/locales/locale.tr.json

```diff
@@ -1,30 +1,58 @@
 {
     "Spec.A_Link": "A Bağlantı",
     "Spec.B_Link": "B Bağlantı",
+    "Spec.Access_Denied": "Erişim Reddedildi",
+    "Spec.Data": "Veri",
     "Spec.Data_1": "Data 1",
     "Spec.Data_2": "Data 2",
     "Spec.Data_3": "Data 3",
     "Spec.Data_4": "Data 4",
+    "Spec.Data_5": "Veri 5",
+    "Spec.Data_6": "Veri 6",
+    "Spec.Data_7": "Veri 7",
+    "Spec.Data_8": "Veri 8",
+    "Spec.Data_9": "Veri 9",
+    "Spec.Data_10": "Veri 10",
     "Spec.Description": "Açıklama",
+    "Spec.Entity": "Varlık",
+    "Spec.Footer_info": "Alt bilgi",
     "Spec.Mid_page": "Orta Sayfa",
+    "Spec.NavLink": "Navigasyon Bağlantısı",
     "Spec.Name": "İsim",
+    "Spec.No_Icon": "Icon Yok",
```

Comment:
`Spec.No_icon`

Reply:
won't repeat key in both locales. do all locale key related comments in both locale files